ogcserver/common.py

```diff
@@ -537,7 +539,7 @@ def xmlhandler(self, code, message, params):
         e.text = message
         if code:
             e.set('code', code)
-        return Response(self.xmlmimetype, ElementTree.tostring(ogcexcetree), status_code=404)
+        return Response(self.xmlmimetype, ElementTree.tostring(ogcexcetree,pretty_print=True), status_code=404)
```

Comment:
@pthorin can you please add the missing space after the comma?

Reply:
```suggestion
        return Response(self.xmlmimetype, ElementTree.tostring(ogcexcetree, pretty_print=True), status_code=404)
```